website/common/locales/en/character.json

```diff
@@ -112,6 +112,7 @@
     "mystery": "Mystery",
     "changeClass": "Change Class, Refund Attribute Points",
     "lvl10ChangeClass": "To change class you must be at least level 10.",
+    "invalidClass":"Invalid class. Please specify warrior, rogue, wizard, or healer.",
```

Comment:
Suggestion: maybe put quotation marks or backticks around the acceptable values?